lms/djangoapps/courseware/access.py

```diff
@@ -466,12 +466,6 @@ def _has_group_access(descriptor, user, course_key):
     This function returns a boolean indicating whether or not `user` has
     sufficient group memberships to "load" a block (the `descriptor`)
     """
-    if len(descriptor.user_partitions) == len(get_split_user_partitions(descriptor.user_partitions)):
```

Comment:
Hopefully this doesn't cause a noticeable performance issue.  We *really* need to transition over to using (the optimized) transformers soon for all course access control.